include/openssl/hmac.h

```diff
@@ -250,12 +250,17 @@ OPENSSL_EXPORT int HMAC_CTX_copy(HMAC_CTX *dest, const HMAC_CTX *src);
 // Private functions
 typedef struct hmac_methods_st HmacMethods;
 
-// We use a union to ensure that enough space is allocated and never actually bother with the named members.
+// We use a union to ensure that enough space is allocated and never actually
+// bother with the named members. We do not externalize SHA3 ctx definition,
```

Comment:
Should we mention that SHA3 ctx is referring to `KECCAK1600_CTX` here? It doesn't seem immediately obvious.

Reply:
I've intentionally kept mentions of `KECCAK1600_CTX` out of the public headers.